rust/datafusion/src/datasource/datasource.rs

```diff
@@ -24,6 +24,24 @@ use crate::arrow::datatypes::SchemaRef;
 use crate::error::Result;
 use crate::physical_plan::ExecutionPlan;
 
+/// The table statistics
+pub struct Statistics {
```

Comment:
The Parquet crate calls these variables. I think consistency is good so maybe they should be renamed?
    num_rows: i64,
    total_byte_size: i64,

Reply:
thanks Mike  I am a new to ruster too, I was fixed the code address comments by you.

Reply:
OK, but I think this maybe num_rows: i64,
total_byte_size: i64,

Reply:
that code is the definition from the Parquet statistics so you should use i64

Reply:
OK I  change the `usize` to `i64`

Reply:
me too